eZ/Publish/Core/Repository/UserService.php

```diff
@@ -80,7 +80,7 @@ public function __construct(RepositoryInterface $repository, Handler $userHandle
             'defaultUserPlacement' => 12,
             'userClassID' => 4,// @todo Rename this settings to swap out "Class" for "Type"
             'userGroupClassID' => 3,
-            'hashType' => User::PASSWORD_HASH_MD5_USER,
+            'hashType' => User::PASSWORD_BCRYPT,
```

Comment:
Should this be named `PASSWORD_HASH_BCRYPT`, so it's consistent with other constant names?